includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -216,13 +216,11 @@ class DesignSystemGlobalFooterModel extends WikiaModel {
 		]
 	];
 
-	private $wikiId;
-	private $lang;
+	protected $lang;
 
-	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
+	public function __construct( $id, $lang = self::DEFAULT_LANG ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)